examples/functions/error/ErrorResponse_1.xml

```diff
@@ -1,59 +1,59 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<siri:OJP xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:siri="http://www.siri.org.uk/siri" xmlns:ojp="http://www.vdv.de/ojp" version="1.1-dev" xsi:schemaLocation="http://www.siri.org.uk/siri ../../../OJP.xsd">
-	<siri:OJPResponse>
+<OJP xmlns="http://www.vdv.de/ojp" xmlns:siri="http://www.siri.org.uk/siri" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.1-dev" xsi:schemaLocation="http://www.vdv.de/ojp ../../../OJP.xsd">
+	<OJPResponse>
 		<siri:ServiceDelivery>
 			<siri:ResponseTimestamp>2022-12-01T17:54:38Z</siri:ResponseTimestamp>
 			<siri:ProducerRef>OJPCH_Prod</siri:ProducerRef>
 			<siri:Status>true</siri:Status>
-			<ojp:OJPLineInformationDelivery>
+			<OJPLineInformationDelivery>
 				<siri:ResponseTimestamp>2022-12-01T17:54:38Z</siri:ResponseTimestamp>
 				<siri:RequestMessageRef>4711</siri:RequestMessageRef>
 				<siri:Status>true</siri:Status>
 				<siri:ErrorCondition>
-					<ojp:OJPError>
+					<OJPError>
 						<siri:ErrorText>Error text</siri:ErrorText>
-						<ojp:OJPGenericProblemType>OJPGENERIC_DATAFRAMEREFNOTAVAILABLE</ojp:OJPGenericProblemType>
-					</ojp:OJPError>
+						<OJPGenericProblemType>OJPGENERIC_DATAFRAMEREFNOTAVAILABLE</OJPGenericProblemType>
+					</OJPError>
 				</siri:ErrorCondition>
-				<!-- no CalcTime for this service <ojp:CalcTime>20</ojp:CalcTime> -->
-				<ojp:LineResult>
-					<ojp:ErrorCondition>
+				<!-- no CalcTime for this service <CalcTime>20</CalcTime> -->
```

Comment:
Should these be removed?

Reply:
now. yes... perhaps you are a precog ;-)